### PR TITLE
NAS-100260 Updated intro to include the new quota exceeded on dataset option.

### PR DESCRIPTION
**Pull request by a-stjohn**

### CODE REVIEW

File: userguide/storage.rst [hunk 1]
```diff
@@ -890,11 +890,30 @@ configure the system to always display advanced settings by enabling the
    +--------------------------+---------------------+---------------+-----------------------------------------------------------------------------------------------------------+
    | Quota for this dataset   | integer             | ✓             | Default of *0* disables quotas. Specifying a value means to use no more than the specified size and is    |
    |                          |                     |               | suitable for user datasets to prevent users from hogging available space.                                 |
+   +--------------------------+---------------------+---------------+-----------------------------------------------------------------------------------------------------------+
+   | Quota warning            | integer             | ✓             | Only available in :guilabel:`Advanced Mode`.                                                              |
```
Comment (wblock) [inline userguide/storage.rst]: Don't use text when the checkbox is available.

File: userguide/storage.rst [hunk 2]
```diff
@@ -902,11 +902,26 @@ configure the system to always display advanced settings by enabling the
    +--------------------------+---------------------+---------------+-----------------------------------------------------------------------------------------------------------+
    | Quota for this dataset   | integer             | ✓             | Default of *0* disables quotas. Specifying a value means to use no more than the specified size and is    |
    |                          |                     |               | suitable for user datasets to prevent users from hogging available space.                                 |
+   +--------------------------+---------------------+---------------+-----------------------------------------------------------------------------------------------------------+
+   | Quota warning            | integer             | ✓             | Show an alert when the dataset quota reaches the specifed value in percent.                               |
```
Comment (wblock) [inline userguide/storage.rst]: Title caps on field name. Also, can't this field just be called "Quota Alert, Percent"?

File: userguide/storage.rst [hunk 3]
```diff
@@ -902,11 +902,26 @@ configure the system to always display advanced settings by enabling the
...
+   | alert at, %              |                     |               | Leave blank to inherit parent dataset values, or enter *0* to disable.                                    |
+   |                          |                     |               |                                                                                                           |
+   +--------------------------+---------------------+---------------+-----------------------------------------------------------------------------------------------------------+
+   | Quota critical           | integer             | ✓             | Show a critical alert when the dataset quota reaches the specified value in percent.                      |
```
Comment (wblock) [inline userguide/storage.rst]: As above, "Quota Critical, Percent".

File: userguide/storage.rst [hunk 4]
```diff
@@ -902,11 +902,26 @@ configure the system to always display advanced settings by enabling the
...
    |                          |                     |               |                                                                                                           |
    +--------------------------+---------------------+---------------+-----------------------------------------------------------------------------------------------------------+
    | Quota for this dataset   | integer             | ✓             | A specified value applies to both this dataset and any child datasets.                                    |
    | and all children         |                     |               |                                                                                                           |
    +--------------------------+---------------------+---------------+-----------------------------------------------------------------------------------------------------------+
+   | Quota warning            | integer             | ✓             | Show an alert when the dataset quota reaches the specifed value in percent.                               |
```
Comment (wblock) [inline userguide/storage.rst]: This and the next field are duplicates of the two fields above.

File: userguide/storage.rst (hunk 2 above)
Comment (a-stjohn) [reply userguide/storage.rst]: This is how it appears in the UI.

File: userguide/storage.rst (hunk 3 above)
Comment (a-stjohn) [reply userguide/storage.rst]: This is how it appears in the UI

File: userguide/storage.rst (hunk 4 above)
Comment (a-stjohn) [reply userguide/storage.rst]: This is how it appears in the UI

**Comment by dlavigne** [comment]
This was already doc'd for release in https://github.com/freenas/freenas-docs/pull/495/files. Does this PR add any additional value? At the very least, the intro changes won't be needed.

**Comment by a-stjohn** [comment]
> This was already doc'd for release in https://github.com/freenas/freenas-docs/pull/495/files. Does this PR add any additional value? At the very least, the intro changes won't be needed.

@dlavigne This one adds information about new fields that were added. In particular, `Quota for this dataset`, `Quota warning alert at, %`, `Quota critical alert at, %`

**Pull request merged**